Build on macOS Ventura is broken

**Environment info:**

* KrakenD version: 2.3.2
* System info: macOS Ventura 13.3.1 (a)
* Hardware specs: Verified on both amd64 and arm64

**Describe the bug**
The build of krakend fails on the most recent Ventura macOS.

**Commands used**
```
$ make build
Building the binary...
# github.com/krakendio/krakend-ce/v2/cmd/krakend-ce
usage: link [options] main.o
[...]
make: *** [build] Error 2
```

**Expected behavior**
Build should succeed.

**Analysis**
Apple has recently and for the first time released a [Rapid Security Response](https://support.apple.com/en-us/HT201224) update. This update adds a letter to the operating system version number: `13.3.1 (a)`.
This breaks `find_glibc.sh` because `sw_vers` now returns the following:
```
ProductName:		macOS
ProductVersion:		13.3.1
ProductVersionExtra:	(a)
BuildVersion:		22E772610a
```
There are two issues now:
1. There are two tabs between `ProductVersion` and `13.3.1`, leading to an empty string being cut out
2. `ProductVersionExtra` also matches the grep and a second line with `(a)` is extracted   

This leads to the script returning
```
DARWIN-
(a)
```
which in turn breaks the build when this is passed to go via `-ldflags`.

--- a/find_glibc.sh
+++ b/find_glibc.sh
@@ -19,7 +19,7 @@ Linux*)
     fi
     ;;
 Darwin*)
-    GLIBC=DARWIN-$(sw_vers | grep ProductVersion | cut -d$'\t' -f2)
+    GLIBC=DARWIN-$(sw_vers | grep ProductVersion: | tr -s '\t' | cut -d$'\t' -f2)
     ;;
 *)
   ;;

